diff --git a/multistep.module b/multistep.module
index cdd72f2..b862f29 100644
--- a/multistep.module
+++ b/multistep.module
@@ -159,13 +159,16 @@ function multistep_block_view($type = '') {
   if (!is_multistep($type)) return;
 
   $router_item = menu_get_item();
-  $page_callback = $router_item['page_callback']; 
+  $page_callback = $router_item['page_callback'];
   
-  if ($page_callback == 'node_page_edit') {
+  $node_edit = array('node_page_edit', 'page_manager_node_edit');
+  $node_add = array('node_add', 'page_manager_node_edit');
+  
+  if (in_array($page_callback, $node_edit)) {
     $node = menu_get_object();
     if ($node->type != $type) return;
     $nid = $node->nid;
-  } elseif($page_callback == 'node_add') {
+  } elseif(in_array($page_callback, $node_add)) {
     if ($router_item['page_arguments'][0] != $type) return;
     $nid = 0;
   } else return;
